### ISSUE TITLE
Writing Console.Ou makes autocomplete go bananas

### ISSUE THREAD

**Issue by erikbra** [dotnet/try#4]
Opening on my mobile (Android, Samsung Galaxy S7).

Wrote Console.Ou in the editor.
Autocompleted to Console.OuConsole.OuConsole.Ou

**Comment by erikbra** [comment]
It's actually

Console.OConsole.OuConsole.Ou

**Comment by piotrpMSFT** [comment]
@erikbra at which URL did you see this behavior? We've not actually enabled autocompletion on the public site yet, but perhaps Monaco is offering some help? I'd like to investigate.

**Comment by erikbra** [comment]
[https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world#step-1](https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world#step-1)

Using standard Chrome on Android
Doesn't happen on Chrome on Windows.

Seems to be a GUI bug, as the complete doesn't make sense (no actual C# logic involved in the autocomplete).  Seems to happen when you press the second key after a `.` (dot).

`a.bn` -> `a.ba.bna.bn`
`longword.fu` -> `longword.flongword.fulongword.fu`

**Comment by piotrpMSFT** [comment]
@erikbra I haven't found a repro device yet. @rchande is working on a completions replacement which may just resolve this issue. We'll let you know once it's live!

**Comment by rchande** [comment]
@erikbra We now have Roslyn-powered completions in our editor. Can you see if this fixes the problem for you?

**Comment by erikbra** [comment]
Sorry, exactly the same problem. Can I do anything to troubleshoot more?

**Comment by jonsequitur** [comment]
Completions aren't available yet at https://docs.microsoft.com. @erikbra, you should be able to verify this at https://try.dot.net.

**Comment by erikbra** [comment]
Yeah, so they tell me. try.dot.net is totally unusable on mobile, which is where the problem appears, so I can't check on try.dot.net. I'm not saying this is code completion. I'm just saying what happens. Something types letters in my browser, and it's not me ;)

**Comment by erikbra** [comment]
Why was this closed? It's still an issue.
